start/pom.xml

```diff
@@ -45,6 +45,7 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-vfs2</artifactId>
+      <version>2.3</version>
```

Comment:
This shouldn't be set here. It should be set in the root `pom.xml` file under the `dependencyManagement` section. I see a comment there that it was held back at 2.1 instead of using 2.2 at some point, due to a bug in 2.2. I wonder if those issues have been fixed in 2.3.